L1Trigger/L1TMuonEndCap/src/SectorProcessorLUT.cc

```diff
@@ -42,6 +43,10 @@ void SectorProcessorLUT::read(unsigned pc_lut_version) {
   read_file(coord_lut_path+"th_lut_neighbor.txt",      th_lut_neighbor_);
   read_file(coord_lut_path+"th_corr_lut_neighbor.txt", th_corr_lut_neighbor_);
 
+  std::string cppf_coord_lut_path = "L1Trigger/L1TMuonEndCap/data/cppf_luts/angleScale_v1/";
```

Comment:
Should probably be separately loaded into the L1Trigger/L1TMuon/data directory, right @rekovic ?